### PR TITLE
Pull request dauwhe 3

### PR DESCRIPTION
**Pull request by dauwhe**
I've rewritten the section on manifests.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -480,13 +480,29 @@
...
+                <li>References to alternative manifestations of the publication.</li>
+                <li>Hints to the user agent about how to present the publication.</li>
+                
+                </ul>
+                
+                
+                
+                
+                 <p>The “Web App Manifest”&nbsp;[[web-manifest]], currently being developed at W3C, is one example of a technology that could define a PWP manifest format. Although the current&nbsp;[[web-manifest]] draft is geared towards Web Applications, it also includes extension points to add information necessary for PWPs. Some of these extra manifest members have already been explored in the <a href="https://github.com/dauwhe/epub31-bff">“Browser Friendly Format”</a> work of the EPUB3.1 Working Group. Given the broad range of uses we envision for <a>Web Publications</a>, there may be several alternatives for serializing the manifest, including EPUB’s package file. </p>
+
```
Comment (iherman) [inline index.html]: Do we want to refer to EPUB at this point? Maybe the last remark could go. (It may raise too many eyebrows...)

**Comment by iherman** [comment]
I like it! Thanks.

File: index.html (hunk 1 above)
Comment (dauwhe) [reply index.html]: I can make it less explicit :) But I don't want to completely dismiss the idea that a possible serialization of a Web Publication may end up looking a lot like EPUB.

**Comment by iherman** [comment]
---
Ivan Herman
Tel:+31 641044153
http://www.ivan-herman.net

(Written on mobile, sorry for brevity and misspellings...)

> @dauwhe commented on this pull request.
> 
> In index.html:
> 
> > +                
> +                <ul>
> +                <li>Publication metadata, directly or by reference.</li>
> +                <li>A list of publication resources, and information about them.</li>
> +                <li>The default sequencing of primary publication components.</li>
> +                <li>References to tables of contents, glossaries, and other navigational or pedagogical aids.</li>
> +                <li>References to alternative manifestations of the publication.</li>
> +                <li>Hints to the user agent about how to present the publication.</li>
> +                
> +                </ul>
> +                
> +                
> +                
> +                
> +                 <p>The “Web App Manifest”&nbsp;[[web-manifest]], currently being developed at W3C, is one example of a technology that could define a PWP manifest format. Although the current&nbsp;[[web-manifest]] draft is geared towards Web Applications, it also includes extension points to add information necessary for PWPs. Some of these extra manifest members have already been explored in the <a href="https://github.com/dauwhe/epub31-bff">“Browser Friendly Format”</a> work of the EPUB3.1 Working Group. Given the broad range of uses we envision for <a>Web Publications</a>, there may be several alternatives for serializing the manifest, including EPUB’s package file. </p>
> +                 
> I can make it less explicit :) But I don't want to completely dismiss the idea that a possible serialization of a Web Publication may end up looking a lot like EPUB.
> 

Don't we talk about this in the section on profiling? If not, isn't that a better place?

**Pull request merged**